tests/G36/data/G36_library_verification_cases.json

```diff
@@ -0,0 +1,176 @@
+{
+    "cases": [
+        {
+            "no": 1,
+            "run_simulation": false,
+            "simulation_IO": {
+                "idf": "./tests/G36/data/G36ReturnAirDamperPositionForReliefDamperOrFan.csv",
```

Comment:
Setting the csv at "idf" probably will not work. We use "output" to load the csv. Are these cases tested?

Reply:
Good point! Addressed.